content/docs/3.0/tasks/deploy/introduction.md

```diff
@@ -7,55 +7,66 @@ menu:
+{{< /info >}}
 
-> **Note:**
-> 
-> - For Oracle Enterprise Linux, TiKV supports the Red Hat Compatible Kernel (RHCK) and does not support the Unbreakable Enterprise Kernel provided by Oracle Enterprise Linux.
-> - A large number of TiKV tests have been run on the CentOS 7.3 system, and in our community there are a lot of best practices in which TiKV is deployed on the Linux operating system. Therefore, it is recommended to deploy TiKV on CentOS 7.3 or later.
-> - The support for the Linux operating systems above includes the deployment and operation in physical servers as well as in major virtualized environments like VMware, KVM and XEN.
+For your **first steps** into TiKV requires only a few things:
 
-## Server requirements
+* A modest machine that fulfills the [system requirements](#system-requirements).
+* A running [Docker](https://docker.com) service.
 
-You can deploy and run TiKV on the 64-bit generic hardware server platform in the Intel x86-64 architecture. The requirements and recommendations about server hardware configuration for development, test and production environments are as follows:
+If you have those, follow through the [`docker-compose`](../docker-compose) guide to get a test setup of TiKV running on your machine.
 
-### Development and test environments
+**Production** usage is typically done via automation requiring:
 
-| Component | CPU     | Memory | Local Storage  | Network  | Instance Number (Minimum Requirement) |
-| :------: | :-----: | :-----: | :----------: | :------: | :----------------: |
-| PD      | 4 core+   | 8 GB+  | SAS, 200 GB+ | Gigabit network card | 1    |
-| TiKV    | 8 core+   | 32 GB+  | SAS, 200 GB+ | Gigabit network card | 3       |
-|         |         |         |              | Total Server Number |  4      |
+* A control machine (it can be one of your target servers) with [Ansible](https://www.ansible.com/) installed.
+* Several (6+) machines fulfilling the [system requirements](#system-requirements) and at least up to [production specifications](#production-specifications).
+* The ability to configure your infrastructure to allow the ports from [network requirements](#network-requirements).
 
-> **Note**: 
-> 
-> - Do not deploy PD and TiKV on the same server.
-> - For performance-related test, do not use low-performance storage and network hardware configuration, in order to guarantee the correctness of the test result.
+If you have these, follow through the [Ansible deployment](../ansible) guide. You may optionally choose unsupported manual [Docker deployment](../docker) or [binary deployment](../binary) strategies.
 
-### Production environment
+Finally, if you want to **develop** TiKV you should consult the [README](https://github.com/tikv/tikv/blob/master/README.md) of the repository.
```

Comment:
I am not sure whether we should include this line - there seems to be no direct relation between TiKV development and TiKV deployment.

Reply:
Let's change it to say "If you want to build your own version of TiKV" as folks may want builds using different feature flags which we don't distribute.